docs/api/README.md

```diff
@@ -0,0 +1,302 @@
+## API Overview
+
+## Available endpoints
+Major resource endpoints supported by the Arrebol API are:
+
+| resource      | description                       |
+|:--------------|:----------------------------------|
+| `/queues`    | manages queue creation and status
+| `/jobs`      | manages job execution and tracking
+| `/workers`   | manages the addition of workers |
+
+### 1 - Queues
+
+#### 1.1 - Create a new queue
+
+**URL**
+```http
+POST /queues
+```
+
+**Request example**
+
+*Body*
+```json
+{
+   "name":"some_awesome_name",
+   "worker_nodes": [
```

Comment:
how about renaming `worker_nodes` to `worker_pool`. Also, rename `worker_pool` to `pool_size`

Reply:
`worker_pools`

Reply:
Ok.